api/base/settings/defaults.py

```diff
@@ -51,21 +51,24 @@
 }
 
 REST_FRAMEWORK = {
-    'PAGE_SIZE': 10,
-
-    # Order is important here because of a bug in rest_framework_swagger. For now,
```

Comment:
The comment was referring to the order of the first two renderers in this list, neither of which are being used now:
```
     'DEFAULT_RENDERER_CLASSES': (
        'rest_framework.renderers.JSONRenderer',		    
        'api.base.renderers.JSONAPIRenderer',		
        'rest_framework.renderers.BrowsableAPIRenderer',	
```	         
They have been replaced by,
```
'rest_framework_json_api.renderers.JSONRenderer'
```

The main issue was the Swagger wouldn't work if it used a renderer with media type `application/vnd.api+json`.  I had resolved this before with custom content negotiation.https://github.com/CenterForOpenScience/osf.io/pull/3683/files#r35565855

A more accurate comment is "This renderer does not work with rest_framework_swagger because of the media type."